File: Linguagem de Programação I/Extras/Beecrowd_Extra/1047.cs
using System;

class Program
{
    static void Main()
    {
        string[] inputs = Console.ReadLine().Split(' ');
        int sh = int.Parse(inputs[0]);
        int sm = int.Parse(inputs[1]);
        int eh = int.Parse(inputs[2]);
        int em = int.Parse(inputs[3]);

        int st = sh * 60 + sm;
        int et = eh * 60 + em;
        int d = et - st;
        if (d <= 0)
        {
            d += 24 * 60;
        }
        int h = d / 60;
        int m = d % 60;
        Console.WriteLine($"O JOGO DUROU {h} HORA(S) E {m} MINUTO(S)");
    }
}

File: Linguagem de Programação I/Extras/Beecrowd_Extra/1047.rs
use std::io::{self, BufRead};

fn main() {
    let stdin = io::stdin();
    let line = stdin.lock().lines().next().unwrap().unwrap();
    let times: Vec<i32> = line.split_whitespace().map(|s| s.parse().unwrap()).collect();
    let (sh, sm, eh, em) = (times[0], times[1], times[2], times[3]);

    let mut st = sh * 60 + sm;
    let mut et = eh * 60 + em;
    let mut d = et - st;
    if d <= 0 {
        d += 24 * 60;
    }
    let h = d / 60;
    let m = d % 60;
    println!("O JOGO DUROU {} HORA(S) E {} MINUTO(S)", h, m);
}